tests/serializers/test_model.py

```diff
@@ -456,6 +456,40 @@ def ser_x(self, v: Any, _) -> str:
     assert s.to_python(Model(x=1000)) == {'x': '1_000'}
 
 
+@pytest.mark.skipif(cached_property is None, reason='cached_property is not available')
+def test_field_serializer_cached_property():
+    @dataclasses.dataclass
+    class Model:
+        x: int
+
+        @cached_property
+        def x_formatted(self) -> str:
+            return f'{self.x:_}'
```

Comment:
Can we try making this also a `computed_field` (or as a second test)? I think there are likely edge cases around cached property interaction with dicts so it would be good to check that they don't break in the presence of computed fields.